lib/stream.js

```diff
@@ -114,14 +113,14 @@ export const getSpawnedResult = async (
 			...stdioStreams.map(stdioStream => waitForStreamEnd(stdioStream, processDone)),
 		]);
 	} catch (error) {
-		return Promise.all([
+		spawned.kill();
```

Comment:
For `childProcess.stdin|stdout|stderr` to be destroyed, we do need to ensure the child process is terminated on errors, i.e. this needs to happen before we wait for those streams' completion.